### PR TITLE
docs: Add how to deploy CAREN

### PR DESCRIPTION
**Pull request by jimmidyson**
Also add some more icon bling to the menu and update menu ordering.

### CODE REVIEW

File: docs/content/getting-started/deployment/via-clusterctl.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Add the following to your `clusterctl.yaml` file, which is normally found at
+`${XDG_CONFIG_HOME}/cluster-api/clusterctl.yaml` (or `${HOME}/cluster-api/clusterctl.yaml`). See [clusterctl
+configuration file] for more details. If the `providers` section already exists, add the entry and omit the `providers`
+key from this block below:
+
+```yaml
+providers:
+  - name: "caren"
+    url: "https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/releases/latest/runtime-extension-components.yaml"
```
Comment (deepakm-ntnx) [inline docs/content/getting-started/deployment/via-clusterctl.md:15]: this is cool solution. do we have a PR on clusterctl cli itself for default location to look for similar to this https://github.com/kubernetes-sigs/cluster-api/blob/main/cmd/clusterctl/client/config/providers_client.go#L277

**Review by deepakm-ntnx — APPROVED** [review]

File: docs/content/getting-started/deployment/via-clusterctl.md (hunk 1 above)
Comment (jimmidyson) [reply docs/content/getting-started/deployment/via-clusterctl.md:15]: Not yet, but i'm happy to open one.

File: docs/content/getting-started/deployment/_index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
++++
+title = "Deploying CAREN"
+icon = "fa-solid fa-truck-fast"
++++
+
+CAREN is implemented as a CAPI runtime extension provider, which means it can be deployed alongside all other CAPI
+providers in the same way [using `clusterctl`]({{< ref "via-clusterctl" >}}). However, as CAREN is not yet integrated
+into `clusterctl`, it is necessary to first configure `clusterctl` to know about CAREN before we can deploy it.
+
+Alternatively you can install CAREN [via Helm]({{< ref "via-helm" >}}). Installing via Helm will provide default some
```
Comment (msdolbey) [inline docs/content/getting-started/deployment/_index.md:10]: Late to the party, but maybe next one, just a nit for a comma here:

`Alternatively, you can install CAREN ...`

**Review by msdolbey — COMMENTED** [review]
Late to the party, but 👍

File: docs/content/getting-started/deployment/_index.md (hunk 2 above)
Comment (jimmidyson) [reply docs/content/getting-started/deployment/_index.md:10]: Thanks!

**Pull request merged**